tests/app/questionnaire/routing/test_operators.py

```diff
@@ -76,6 +79,15 @@ def test_operation_greater_than(operands, expected_result):
     assert operator.evaluate(operands) is expected_result
 
 
+@pytest.mark.parametrize(
+    "operands, expected_result",
+    greater_than_and_less_than_operations_equals_test,
+)
+def test_operation_greater_than_same_number(operands, expected_result):
+    operator = Operator(Operator.GREATER_THAN)
+    assert operator.evaluate(operands) is expected_result
```

Comment:
This is already being testing and is not needed.

Reply:
Ive added this test and the one below because for greater than and less than I realised that there was no test for if the number entered was the same as the number in the question. For example `
[(1, 1), False],`. This is only tested for currently on the `<=` and `>=` tests. I also needed to do them separately to the other tests because of the way the other tests were written, less than and greater than expect opposite results but in this case they would both be expecting "False"